pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,131 @@
+---
+title: Project ideas
+video_player: true
+theme: responsive
+---
+<style>
+  table {
+    table-layout: fixed;
+    width: 100%;
+  }
+  td {
+    vertical-align: top;
+    text-align: center;
+  }
+  img {
+    width: 100%;
+  }
+</style>
+<a id="top"></a>
+# Project Ideas for K-5 Elementary School Students
+<p> <a href="https://code.org/educate/spritelab">Sprite Lab</a> can be used by anyone but are targeted at elementary school students. Programming in a brand new environment can be challenging, but you can do it! For more information about full course offerings, check out <a href="https://code.org/educate/curriculum/elementary-school">CS Fundamentals</a> to find the right course for your age and skill level.</p>
```

Comment:
I'm not sure whether a change in how this page is authored is within scope, since presumably it's using an existing format, but I think there is some room for improvement:

- uses a lot of html, and not much markdown
- lines are very long
- not cleanly formatted
- inconsistent indentation widths

I embarked on some new ideas for authoring this kind of page that you're welcome to consider, in PRs associated with [this one](https://github.com/code-dot-org/code-dot-org/pull/42654).  The goal was to have a clean markdown format that could be authored without knowledge of html/css, while still being localized, and with a clean separation from a set of components that could be reused, and maintained independently.

Reply:
I formatted the file so it's much more readable and consistent, but stopped short of converting to markdown and revamping how content editors can update the page for the sake of time and other, higher priorities.  However, I logged a [Jira ticket](https://codedotorg.atlassian.net/browse/LP-2105) reminder to check the page views for the new project ideas page in Google Analytics this spring. If the page is getting solid traffic, we can invest in localizations and more content-editor-friendly formatting then.  Thanks for the feedback and cool ideas about the self-serve components!